cmd/kubeadm/app/apis/kubeadm/v1beta2/types.go

```diff
@@ -23,119 +23,120 @@ import (
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// InitConfiguration contains a list of elements that is specific "kubeadm init"-only runtime
-// information.
+// InitConfiguration contains runtime information that are specific to "kubeadm init".
+// These information are only used the first time `kubeadm init` runs.
```

Comment:
These information is -> This information is

Reply:
Right. Just checked the grammar book, it says "Uncountable nouns are only used in the singular tense."
Thanks. Will fix.